builtin/credential/cert/path_login.go

```diff
@@ -161,6 +161,12 @@ func (b *backend) pathLoginRenew(ctx context.Context, req *logical.Request, d *f
 		return nil, err
 	}
 
+	if b.crls == nil {
+		if err := b.populateCRLs(ctx, req.Storage); err != nil {
```

Comment:
I think, to me, the cleaner design going forward would be to mirror PKI: a dirty atomic boolean and updating it from the periodic func on invalidation; the others just get slightly stale data until this is updated but don't have to worry about `b.crls` potentially being nil and handling the consequences of that. My 2c.

Reply:
Rather than niling it out, good idea.  I'll make this change after getting these fixes backported.

Reply:
Yeah, definitely something I'd only do in 1.12. ty!